ms-oidc/src/main/java/net/trajano/ms/oidc/internal/IssuerConfig.java

```diff
@@ -1,20 +1,22 @@
 package net.trajano.ms.oidc.internal;
 
-import java.io.IOException;
-import java.net.MalformedURLException;
-import java.net.URI;
-import java.nio.charset.StandardCharsets;
-import java.text.ParseException;
-import java.util.Base64;
+import net.trajano.ms.oidc.OpenIdConfiguration;
+import org.jose4j.jwk.HttpsJwks;
+import org.jose4j.jwk.JsonWebKey;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.jose4j.jwk.JsonWebKey'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)